posthog/api/cohort.py

```diff
@@ -121,14 +263,34 @@ class BehavioralFilter(BaseModel, extra="forbid"):
     explicit_datetime: str | None = None
 
 
-class CohortFilter(BaseModel, extra="forbid"):
+class RealtimeFilter(FilterBytecodeMixin, BaseModel, extra="forbid"):
```

Comment:
I have very mixed feelings about this - on one hand we have our own filter - great. on the other we need to migrate everything over which makes it really hard to test with out current load.

atm its identical to a behavioral filter it seems - lets just keep behavorial and use the sub value to decide if it is supported or not